qemu/tests/migration_multi_host_downtime_and_speed.py

```diff
@@ -8,7 +8,6 @@
 from virttest import utils_test
 from virttest import remote
 from virttest import virt_vm
-from virttest import utils_misc
 
 from provider import cpuflags
 
```

Comment:
Could you pls add test description 
    """
    KVM multi-host migration test:
    Migration execution progress is described in documentation
    for migrate method in class MultihostMigration.
1.
2. ..
3. ..